privacyidea/lib/caconnectors/localca.py

```diff
@@ -568,12 +569,14 @@ def create_crl(self, publish=True, check_validity=False):
 
         return ret
 
-    @classmethod
-    def create_ca(cls, name):
+    @staticmethod
+    def create_ca(name):
```

Comment:
I think the original idea was, that if we one day have other CA types like a MS CA, the create method could need some more information, this is why it was created as classmethod. (Well, actually the method is missing in the baseclass.

But the the ``pi-manage ca create -t msca`` could also work.

We can however change this and if necessary change this back, when we create an MSCA connector one day.
What do you think?

Reply:
The ``localca.create_ca`` must not be part of pi-manage. It must be part of the localca module, since each connecter would have it's own logic of creating a CA setup. So this logic needs to be part of the module. Not sure if it needs to be part of the class - probably not.

Currently I do not know, how the creation of a MS CA would look like. No idea, so we should stay flexible enough.

``staticmethod`` is totally fine for me, now.